documents/index.json

```diff
@@ -0,0 +1 @@
+{"signature":"34fbecc5b263e17dba2cd597119489a17b7343d6"}
```

Comment:
What is this?

Reply:
document signature, it's preventing documents from staged going into production or the other way round :)

Reply:
We maybe able to remove this.